src/lambda/commands/createNewSamApp.ts

```diff
@@ -54,6 +54,7 @@ import { execSync } from 'child_process'
 type CreateReason = 'unknown' | 'userCancelled' | 'fileNotFound' | 'complete' | 'error'
 
 export const SAM_INIT_TEMPLATE_FILE: string = 'template.yaml'
+export const SAM_INIT_TEMPLATE_FILE_ALT_EXT: string = 'template.yml'
```

Comment:
Should this not be modeled as a list of items instead adding another special-case variable + parameter?